commit 9296c5bb57078a376576291915f39ff02c08b33e (refs/changes/94/5694/2)
Author: Trent Mick <trent.mick@joyent.com>
Date:   2019-03-01T23:33:05+00:00 (7 months ago)
    
    TRITON-1278 sdc-adminui 'make release' includes whatever is lying around, resulting in yuge images
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/Makefile b/Makefile
index 719c58f8..144669b0 100644
--- a/Makefile
+++ b/Makefile
@@ -113,14 +113,29 @@ test: | $(JS_BUNDLE)
 .PHONY: release
 release: all deps docs $(SMF_MANIFESTS)
 	@echo "Building $(RELEASE_TARBALL)"
-	@mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)
-	@mkdir -p $(RELSTAGEDIR)/site
-	@touch $(RELSTAGEDIR)/site/.do-not-delete-me
-	cp -r $(ROOT)/* $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/
-	rm -rf $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/*.gz
-	rm $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/image-version.json
-	@echo "{\"version\": \"$(STAMP)\"}" >> $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/image-version.json
-	rm -rf $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/deps
+	mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)
+	mkdir -p $(RELSTAGEDIR)/site
+	touch $(RELSTAGEDIR)/site/.do-not-delete-me
+	cp -r \
+		$(ROOT)/bootstrap \
+		$(ROOT)/etc \
+		$(ROOT)/less \
+		$(ROOT)/lib \
+		$(ROOT)/LICENSE \
+		$(ROOT)/node_modules \
+		$(ROOT)/package.json \
+		$(ROOT)/README.md \
+		$(ROOT)/sapi_manifests \
+		$(ROOT)/server.js \
+		$(ROOT)/smf \
+		$(ROOT)/views \
+		$(ROOT)/www \
+		$(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/
+	mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/build
+	cp -PR \
+		$(TOP)/build/node \
+		$(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/build
+	@echo "{\"version\": \"$(STAMP)\"}" > $(RELSTAGEDIR)/root/opt/smartdc/$(NAME)/image-version.json
 	mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/boot
 	cp -R $(ROOT)/deps/sdc-scripts/* $(RELSTAGEDIR)/root/opt/smartdc/boot/
 	cp -R $(ROOT)/boot/* $(RELSTAGEDIR)/root/opt/smartdc/boot/
diff --git a/package.json b/package.json
index 208a9489..92a6807a 100644
--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "author": "Kevin Chan",
   "name": "adminui",
   "description": "Triton Operations Portal",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "private": true,
   "repository": {
     "git": "git@git.github.com:joyent/sdc-adminui.git"
